applications/luci-app-throughput-measurements/root/etc/config/tputmeas

```diff
@@ -0,0 +1 @@
+config iperf3
```

Comment:
The configuration belongs to the luci, doesn't it? Wouldn't it then be better to identify it? The configuration only works with the luci. Can you then also list all the configurations.

Reply:
ok